packages/datatrak-web/src/features/MobileAppPrompt.tsx

```diff
@@ -75,54 +67,95 @@ const BROWSER_ICONS = {
   [BROWSERS.EDGE]: '/edge-icon.png',
 };
 
+// each of these will have a fallback url and an app url. The app url will be a link to the app itself, and the fallback url will be a link to the applicable app store
 const APP_URL = {
-  ANDROID: 'https://play.google.com/store/apps/details?id=com.tupaiameditrak',
-  IOS: 'https://itunes.apple.com/us/app/tupaia-meditrak/id1245053537',
+  ANDROID: {
+    fallback: 'https://play.google.com/store/apps/details?id=com.tupaiameditrak',
+    app: 'intent://#Intent;package=com.tupaiameditrak;end', // eventually this is where the deep link will go
+  },
+  IOS: {
+    fallback: 'https://itunes.apple.com/us/app/tupaia-meditrak/id1245053537',
```

Comment:
Perhaps safer to let the user’s device decide which App Store region to use?

```suggestion
    fallback: 'https://itunes.apple.com/app/tupaia-meditrak/id1245053537',
```

Reply:
Good point!